Libraries/LibHTML/DOM/Document.cpp

```diff
@@ -19,3 +19,18 @@ StyleResolver& Document::style_resolver()
         m_style_resolver = make<StyleResolver>(*this);
     return *m_style_resolver;
 }
+
+void Document::normalize()
```

Comment:
I would call this something else to avoid conflicting with the DOM's Node.normalize():
https://developer.mozilla.org/en-US/docs/Web/API/Node/normalize